src/server/dao/usersDao.ts

```diff
@@ -0,0 +1,39 @@
+import { Client } from 'pg';
+
+export interface DaoConfig {
+    connectionString: string;
+    ssl: boolean;
+}
+
+export class UsersDao {
+    private client: Client;
+
+    constructor(private config: DaoConfig) {
+        this.client = new Client(config);

```

Comment:
I think we can leave this for now but we should probably look into using [Pooling](https://node-postgres.com/features/pooling) and have it defined inside the main routes file or something so we don't need to make several different connections to the db.

Reply:
Certainly agree with the point about pooling for the DAO classes - I made a Trello card for it under the Grooming List.